scss/tech.scss

```diff
@@ -1,3 +1,53 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
+  margin: 20px;
+  padding: 10px 20px 20px 10px;
```

Comment:
Order is `top right bottom left`. Do you want content to be moved top left? If not, `10px 20px` should work

Reply:
It was just to move away the bottom and right a little more like is in the card's image. I set all the padding to 20px now, let me know if you are ok with it.